src/FunderMaps.WebApi/Controllers/Portal/IncidentPortalController.cs

```diff
@@ -72,14 +78,16 @@ public async Task<IActionResult> UploadDocumentAsync([Required] IFormFile input)
             }
 
             // Act.
-            var fileName = await _incidentUseCase.StoreDocumentAsync(
-                input.OpenReadStream(),
-                input.FileName,
-                input.ContentType);
+            var storeFileName = Core.IO.Path.GetUniqueName(input.FileName);
+            await _blobStorageService.StoreFileAsync(
+                containerName: "incident-report",
```

Comment:
I suggest not hard coding this but storing this in some constants file. The constants file for this repo had it's storage constants removed, so lets place them in our core or in a configuration file (if we wan them dynamically).

Reply:
Conclusion: These constants will be placed in our `Core`.